src/api/services/TransactionAPI.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Italia PagoPA Proxy
+ * Cittadinanza Digitale PagoPA services
+ */
+
+import * as express from "express";
+import fetch from "node-fetch";
+import querystring = require("querystring");
+import { CONFIG } from "../../Configuration";
+import { ITransactionListResponse } from "../types/TransactionResponse";
+
+// Transaction Service for PagoPA communications
+export class TransactionAPI {
+  // Retrieve transactions
```

Comment:
please add a little more verbose comments on each method that explain what the method is supposed to do a little bit more in depth for developers that don't have context here (and for code rewievers =)